steward/src/commands/cosmos_mode.rs

```diff
@@ -23,14 +23,20 @@ impl Runnable for CosmosSignerCmd {
         info!("Starting application");
         abscissa_tokio::run(&APP, async {
             // Reflection required for certain clients to function... such as grpcurl
-            let contents = server::get_steward_descriptor_contents().unwrap();
+            let contents = server::DESCRIPTOR.to_vec();
             let proto_descriptor_service = tonic_reflection::server::Builder::configure()
                 .register_encoded_file_descriptor_set(contents.as_slice())
                 .build()
-                .unwrap();
+                .unwrap_or_else(|err| {
+                    panic!("failed to build descriptor service: {}", err);
```

Comment:
From a user's perspective the app will exit in a cleaner fashion if you print out an error message and then call `exit(1)` rather than using `panic!`.

Reply:
fixed